src/app/pages/Article/index.test.jsx

```diff
@@ -33,7 +33,7 @@ jest.mock('../PageHandlers/withPageWrapper', () => Component => {
   return PageWrapperContainer;
 });
 
-jest.mock('../PageHandlers/withLoading', () => Component => {
+jest.mock('../../containers/PageHandlers/withLoading', () => Component => {
```

Comment:
Or better yet, maybe move `PageHandlers` into `pages`, would actually reduce the amount of changes in this PR.

Reply:
I was thinking about moving the page handlers too. Just wasn't 100% sure, but if you think its a good idea I am up for it.

Might be worth doing it in a different PR as to not make this one any bigger. thoughts?

Reply:
I was wrong to say it would reduce the changes but it would be a lot less changes and lot more straight file moves without any paths changing, i.e. this line would no long need to change. 

I'd perhaps think about doing it in this PR in case any path changes were missed in the refactor, getting that sorted before merging would make it less likely we have any regressions. It's not a strong opinion though, maybe a follow-up PR would allow us to discuss the move's merits seperately re @simonsinclair 's point.

Reply:
I would prefer a follow up PR and more discussion as keeping this new dir strictly about page types will be very useful I think.

Reply:
Cool, how do we go about the follow-up, just make the PR or raise an issue etc?

Reply:
can just raise an issue and pick it up.